.goreleaser.yml

```diff
@@ -0,0 +1,45 @@
+# This is an example goreleaser.yaml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod download
+    # you may remove this if you don't need go generate
+    - go generate ./...
+builds:
+  - id: grpcserver
+    binary: grpcserver
+    dir: cmd/server
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    ldflags: -s -w -X sigs.k8s.io/aws-encryption-provider/pkg/version.Version={{.Version}} -X sigs.k8s.io/aws-encryption-provider/pkg/version.Commit={{.Commit}} -X sigs.k8s.io/aws-encryption-provider/pkg/version.Date={{.Date}}
+  - id: grpcclient
```

Comment:
The client is purely for testing, I don't think it makes sense to include it release artifacts